src/main/java/uk/gov/hmcts/reform/idam/api/fr/am/oidc/model/JsonWebKey.java

```diff
@@ -441,7 +451,7 @@ public int hashCode() {
   public String toString() {
     StringBuilder sb = new StringBuilder();
     sb.append("class JsonWebKey {\n");
-    
+
```

Comment:
Accidental new line?

Reply:
Generated code, I'll leave as is because next time it gets generated it will come back